helm/dex-app/templates/dex-k8s-authenticator/configmap-customer.yaml

```diff
@@ -1,4 +1,58 @@
 {{ if .Values.oidc.customer.enabled }}
+{{- if .Values.oidc.customer.connectors -}}
```

Comment:
We create one configmap for each connector given in the new `connectors` list field. If this field is not populated, we use the current setup.